.gitignore

```diff
@@ -43,6 +46,7 @@ htmlcov/
 nosetests.xml
 coverage.xml
 *,cover
+.pytest_cache/
```

Comment:
Seems these are already ignored from git  in VSCode somehow, but yet they weren't specified in .gitignore.